vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -499,26 +548,53 @@ func (e *ProposalTxExecutor) RewardValidatorTx(tx *txs.RewardValidatorTx) error
 
 		// Reward the delegatee here
 		if delegateeReward > 0 {
-			delegationRewardsOwner := vdrTx.DelegationRewardsOwner()
-			outIntf, err := e.Fx.CreateOutput(delegateeReward, delegationRewardsOwner)
-			if err != nil {
-				return fmt.Errorf("failed to create output: %w", err)
+			if vdrStaker.StartTime.After(e.Config.CortinaTime) {
+				previousDelegateeReward, err := e.OnCommitState.GetDelegateeReward(
+					vdrStaker.SubnetID,
+					vdrStaker.NodeID,
+				)
+				if err != nil {
+					return fmt.Errorf("failed to get delegatee reward: %w", err)
+				}
+
+				newDelegateeReward, err := math.Add64(previousDelegateeReward, delegateeReward)
+				if err != nil {
+					return fmt.Errorf("delegatee reward overflowed: %w", err)
```

Comment:
Will this halt execution? Seems like someone could then mint a token with a custom AssetID to hit this.

Reply:
I think we'd be better off setting to max Uint64 instead of erroring?

Reply:
Is that possible? Isn't the max supply of a token capped, meaning the rewards can't exceed MaxUint64?

Reply:
I'm not sure. If not, we should make sure to comment that here IMO.

Reply:
Removed the `Add64` call and documented the invariant assumption.